pkg/constant/constant_windows.go

```diff
@@ -36,6 +36,7 @@ const (
 	KineSocket                     = "kine\\kine.sock:2379"
 	KubePauseContainerImage        = "mcr.microsoft.com/oss/kubernetes/pause"
 	KubePauseContainerImageVersion = "1.4.1"
+	K0sConfigPathDefault           = "C:\\etc\\k0s\\k0s.yaml"
```

Comment:
There's probably a better location for the default config (and rundir) on windows. Something like `path.Join(os.UserConfigDir(), "k0s")`. (which becomes something like `%APPDATA%/k0s`, or `C:\Users\User\AppData\Roaming\K0s`).

I guess some masochistic users might not even have `C:` or maybe a read-only `C:`.

Reply:
> There's probably a better location for the default config (and rundir) on windows. Something like `path.Join(os.UserConfigDir(), "k0s")`. (which becomes something like `%APPDATA%/k0s`, or `C:\Users\User\AppData\Roaming\K0s`).
> 
> I guess some masochistic users might not even have `C:` or maybe a read-only `C:`.

Actually, there is no need at all for this var on windows, since we don't support windows controllers.
That also relates to the entire `pkg/config` package which is not needed at all in the windows binary. Unfortunately, in the current architecture, excluding the package from the windows build turned out to be a much bigger change than anticipated, and maybe should be tackled in a separate PR.